test/evergreen.yml

```diff
@@ -132,32 +132,76 @@ functions:
         set -o errexit
         set -o verbose
         for i in $(seq ${times|1}); do
-          ./t -1 -c ${config|../../../test/format/CONFIG.stress} ${extra_args|}
+          ./t -1 -c ${config|../../../test/format/CONFIG.stress} ${extra_args|} 2>&1
         done
   "many dbs test":
     command: shell.exec
-    parms:
+    params:
```

Comment:
There are few typos for `parms` -> `params` that I am fixing using this ticket.

Reply:
Good catch. I'm surprised `evergreen validate` did not pick it out.

Reply:
Yes I was surprised as well and I only picked it once I realized that some of sub-tasks were returning within few microseconds.